docs/user_guide/admission_controller.md

```diff
@@ -0,0 +1,200 @@
+  # Schedule only pods in spark-* and mpi-* namespaces with YuniKorn
+  admissionController.filtering.processNamespaces: "^spark-,^mpi-"
+  # Don't schedule pods in kube-system or fluentd-* namespaces
+  admissionController.filtering.bypassNamespaces: "^kube-system$,^fluentd-"
+  # Add applicationId labels to pods spark-* namespaces
+  admissionController.filtering.labelNamespaces: "^spark-"
+  # Don't add applicationId labels to pods noqueue namespaces
+  admissionController.filtering.noLabelNamespaces: "^noqueue$"
+  admissionController.filtering.generateUniqueAppId: "false"
+  # Change default queue to root.mydefault
+  admissionController.filtering.defaultQueue: "root.mydefault"
+  admissionController.accessControl.bypassAuth: "false"
+  admissionController.accessControl.trustControllers: "true"
+  admissionController.accessControl.systemUsers: "^system:serviceaccount:kube-system:"
+  admissionController.accessControl.externalUsers: ""
+  admissionController.accessControl.externalGroups: ""
+  queues.yaml: |
+    partitions:
+      - name: default
+        placementrules:
+          - name: tag
+            value: namespace
+            create: true
+        queues:
+          - name: root
+            submitacl: '*'
+  
+```
+Use the following command to create ConfigMap to change settings.
+```shell script
+kubectl apply -f yunikorn-configs.yaml -n yunikorn
+```
+Use the following command to check whether the ConfigMap is created successfully.
+```shell script
+kubectl get ConfigMap -n yunikorn
+```
+Seeing that yunikorn-configs represents successful creation
+```shell script
+NAME				DATA	AGE
+yunikorn-configs	14		10s
+yunikorn-defaults   0		1d
+```
+
+### Use case
+The following are the usage scenarios of using admission controller to set up in different deployment modes of Yunikorn.
+
+For more information on Yunikorn's deployment modes, please refer to [Deployment modes](deployment_modes.md).
+#### Standard deployment, everything scheduled by Yunikorn
+In standard deployment mode, can set which namespaces will be scheduled by YuniKorn through the `admissionController.filtering.processNamespaces` tag in `yunikorn-config`.
+
+If want all namespaces to be scheduled by YuniKorn, just set `admissionController.filtering.processNamespaces` to the empty string (default).
+
+```yaml
+#All namespaces will be schduler by YuniKorn
+admissionController.filtering.processNamespaces: ""
+```
+
+#### Plugin deployment, everything scheduled by YuniKorn
+Same as the standard mode, can set which namespaces will be scheduled by YuniKorn through the `admissionController.filtering.processNamespaces` tag in `yunikorn-config`.
+
+If want all namespaces to be scheduled by YuniKorn, just set `admissionController.filtering.processNamespaces` to the empty string (default).
+
+```yaml
+#All namespaces will be schduler by YuniKorn
+admissionController.filtering.processNamespaces: ""
+```
+#### Plugin deployment, only workloads in some namespaces scheduled byYuniKorn
```

Comment:
Typo: "byYunikorn"

I think the "only" is not needed here (sounds weird to me).